src/Microsoft.Health.Fhir.Api.UnitTests/Features/Audit/AuditLoggingFilterAttributeTests.cs

```diff
@@ -77,7 +78,7 @@ public void GivenAController_WhenExecutedAction_ThenAuditLogShouldBeLogged()
         {
             const HttpStatusCode expectedStatusCode = HttpStatusCode.Created;
 
-            var fhirResult = new FhirResult(new Patient() { Name = { new HumanName() { Text = "TestPatient" } } });
+            var fhirResult = new FhirResult(new Patient() { Name = { new HumanName() { Text = "TestPatient" } } }.ToResourceElement());
```

Comment:
Is there a way to directly deserialize into ITypedElement without having to deserialize into a resource first?

Reply:
I'll keep looking into this